src/main/kotlin/com/arabyte/arabyteapi/domain/user/controller/UserController.kt

```diff
@@ -0,0 +1,24 @@
+package com.arabyte.arabyteapi.domain.user.controller
+
+import com.arabyte.arabyteapi.domain.user.service.UserService
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/user")
+class UserController(
+    private val userService: UserService
+) {
+
+    @PostMapping("/withdraw")
```

Comment:
```suggestion
    @DeleteMapping
```

post를 사용하는 이유가 있나요??

Reply:
@DeleteMapping 변경해서 Http 메서드로 구분하는 방법이 훨씬 좋아보입니다. 기능도 사실상 delete하는 것 외에는 없어서 Post는 별로네요